From 88491e3ed7db83d06847ac7372bc4345dd92d4ee Mon Sep 17 00:00:00 2001
From: bagyusz <phelyx92@gmail.com>
Date: Tue, 18 Nov 2014 17:21:56 +0100
Subject: [PATCH] [REVERT] mm-video: venc: Correct a typo in variable name A 
 kernel variable was to be defined as unsigned long but it is mistakenly 
 defined as unsigned only, the space is missing after long. This bug is silent
  because unsigned is also a valid data type by itself. Corresponding to
 kernel  fix, similar correction is done in userspace code.

Change-Id: Ie7518197d3494ba0ea362c264fec98f85431cd4f
---
 mm-video/vidc/venc/src/video_encoder_device.cpp | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/mm-video/vidc/venc/src/video_encoder_device.cpp b/mm-video/vidc/venc/src/video_encoder_device.cpp
index 96faccc..a66f262 100644
--- a/mm-video/vidc/venc/src/video_encoder_device.cpp
+++ b/mm-video/vidc/venc/src/video_encoder_device.cpp
@@ -1613,7 +1613,7 @@ void venc_dev::venc_config_print()
                    multislice.mslice_size);
 
   DEBUG_PRINT_HIGH("\nENC_CONFIG: EntropyMode: %d, CabacModel: %d",
-                   entropy.entropysel, entropy.cabacmodel);
+                   entropy.longentropysel, entropy.cabacmodel);
 
   DEBUG_PRINT_HIGH("\nENC_CONFIG: DB-Mode: %d, alpha: %d, Beta: %d\n",
                    dbkfilter.db_mode, dbkfilter.slicealpha_offset,
@@ -2447,7 +2447,7 @@ bool venc_dev::venc_set_entropy_config(OMX_BOOL enable, OMX_U32 i_cabac_level)
   DEBUG_PRINT_LOW("\n venc_set_entropy_config: CABAC = %u level: %u", enable, i_cabac_level);
 
   if(enable &&(codec_profile.profile != VEN_PROFILE_H264_BASELINE)){
-    entropy_cfg.entropysel = VEN_ENTROPY_MODEL_CABAC;
+    entropy_cfg.longentropysel = VEN_ENTROPY_MODEL_CABAC;
       if (i_cabac_level == 0) {
          entropy_cfg.cabacmodel = VEN_CABAC_MODEL_0;
       }
@@ -2467,7 +2467,7 @@ bool venc_dev::venc_set_entropy_config(OMX_BOOL enable, OMX_U32 i_cabac_level)
 #endif
   }
   else if(!enable){
-    entropy_cfg.entropysel = VEN_ENTROPY_MODEL_CAVLC;
+    entropy_cfg.longentropysel = VEN_ENTROPY_MODEL_CAVLC;
     }
   else{
     DEBUG_PRINT_ERROR("\nInvalid Entropy mode for Baseline Profile");
@@ -2481,7 +2481,7 @@ bool venc_dev::venc_set_entropy_config(OMX_BOOL enable, OMX_U32 i_cabac_level)
     DEBUG_PRINT_ERROR("\nERROR: Request for setting entropy config failed");
     return false;
   }
-  entropy.entropysel = entropy_cfg.entropysel;
+  entropy.longentropysel = entropy_cfg.longentropysel;
   entropy.cabacmodel  = entropy_cfg.cabacmodel;
   return true;
 }
-- 
1.9.2

